docs/ocis/release_notes.md

```diff
@@ -7,6 +7,44 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
 
+## ownCloud Infinite Scale 1.20.0 Technology Preview
+
+Version 1.20.0 brings major improvements, new features and bug fixes to the platform. Infinite Scale now provides complete Auditing capabilities and the basic Spaces' feature has reached initial feature completeness. Furthermore, ownCloud Web introduces a number of smaller features as well as more design and usability improvements.
+
+The most prominent changes in ownCloud Infinite Scale 1.20.0 and ownCloud Web 5.4.0 comprise:
+
+* The implementation of the basic 'Auditing' feature is now complete. [ocis#3467](https://github.com/owncloud/ocis/pull/3467)
+* The implementation of the basic 'Spaces' feature is now complete. [web#6693](https://github.com/owncloud/web/pull/6693) [web#6659](https://github.com/owncloud/web/pull/6659) [web#6639](https://github.com/owncloud/web/pull/6639) [web#6633](https://github.com/owncloud/web/pull/6633) [web#6662](https://github.com/owncloud/web/pull/6662) [web#6642](https://github.com/owncloud/web/pull/6642)
+* All Space members can now list all links and shares. [ocis#3370](https://github.com/owncloud/ocis/issues/3370)
+* The LDAP configuration settings have been simplified and unified across services. [#3476](https://github.com/owncloud/ocis/pull/3476)
+* All sharing options (users & links) are now united in one panel in ownCloud Web. [web#6701](https://github.com/owncloud/web/pull/6701)
+* The "Media Viewer" in ownCloud Web has been renamed to "Preview". [web#6514](https://github.com/owncloud/web/pull/6514)
+* ownCloud Web now has support for audio playback in "Preview" (e.g., MP3, WAV, FLAC, OGG). [web#6514](https://github.com/owncloud/web/pull/6514)
+* Public links on single files can now be created with the Editor role. [#6618](https://github.com/owncloud/web/pull/6618)
```

Comment:
Did you double check if this really works with `vanilla oCIS`? afair this was EOS related...

Reply:
Let me check tomorrow. I have seen an ocis backend ticket for that.

Reply:
So, finally the feature is not part of vanilla oCIS, but will be switched on in the next release

Reply:
```suggestion
```